docs/01-Introduction.md

```diff
@@ -1,5 +1,55 @@
-# 1.0 - Introduction
+# 1.0 - Introduction 
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI?
+LORIS-MRI is the backbone of the imaging component that makes up LORIS. 
+These documents assume you have some
+knowledge regarding LORIS and a functioning installation. For information
+regarding LORIS itself, please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is responsible for the processing, visualizing, and archiving
```

Comment:
This is seriously great.  
I would suggest specifying _what_ it is: 
- a separate repo, so that it can be installed on the file server not the web server
- a set of libraries and scripts that handle imaging file formats, pre-processing, quality checks and insertion. (etc, then go on to as you said what it's responsible for...)
- not necessarily limited to MRI (? already/soon)

Reply:
@davidblader 

maybe you can incorporate what @christinerogers mentioned above? 
I would personally not document the third item (it should be updated in the pull request that adds more than MRI support).

Reply:
Personally, I would replace the first sentence by:
LORIS-MRI is responsible for the insertion, visualization and archival of uploaded imaging datasets.

Two reasons:
1) no processing happens in LORIS, just conversion and insertion. Processing happens in CBRAIN or using other tools.
2) modifying "MRI scans" to "Imaging dataset" here as soon, we will support more than just MRI datasets.